smack-openpgp/src/test/java/org/jivesoftware/smackx/ox/SecretKeyBackupHelperTest.java

```diff
@@ -51,7 +52,7 @@ public class SecretKeyBackupHelperTest extends SmackTestSuite {
     private static final File basePath;
 
     static {
-        basePath = new File(org.apache.commons.io.FileUtils.getTempDirectory(), "ox_secret_keys");
+        basePath = new File(org.apache.commons.io.FileUtils.getTempDirectory(), "ox_secret_keys_" + StringUtils.randomString(10));
```

Comment:
Let's do the idiomatic thing:

```suggestion
        basePath = Files.createTempDirectory("ox_secret_keys").toFile();
```

Reply:
Could we please move this PR along so that we can get ignite's bamboo jobs green again, thanks.